qiita_pet/templates/redbiom.html

```diff
@@ -153,21 +158,83 @@
 {%block content%}
   <small>
     Redbiom only searches on public data and the data is updated nightly. Note that you will only be able to expand and add artifacts to analyses if you are signed into Qiita.
+    <br/><br/>
+    <button class="btn btn-info btn-sm" data-toggle="collapse" data-target="#redbiom-help">Help and examples?</button>
+    <br/>
+  </small>
+  <div id="redbiom-help" class="collapse">
     <br/>
     We have 3 search options:
     <ul>
-      <li><b>Metadata</b>: The search will be on the full metadata and will return any ocurrances of the search.</li>
-      <li><b>Feature</b>: The seach will be on all the features, in specific: OTU ids for close reference and exact sequences for deblur</li>
-      <li><b>Taxon</b>: The search will be only on close reference and based on the taxonomies available</li>
+      <li>
+        <b>Metadata</b><br/>
+        The search will be on the full metadata.
+        <br/><br/>
+        The metadata search engine uses natural language processing to search for
+        word stems within a samples metadata. A word stem disregards modifiers and
+        plurals, so for instance, a search for "antibiotics" will actually perform
+        a search for "antibiot". Similarly, a search for "crying" will actually
+        search for "cry". The words specified can be combined with set-based
+        operations, so for instance, a search for "antibiotics &amp; crying" will
+        obtain the set of samples in which each sample has "antibiot" in its
+        metadata as well as "cry". N.B., the specific category in which a stem is
+        found is not assured to be the same, "antibiot" could be in one category
+        and "cry" in another. A set intersection can be performed with "&amp;", a
+        union with "|" and a difference with "-".
+        <br/><br/>
+        In addition to the stem-based search, value based searches can also be a
+        applied. These use a Python-like grammar and allow for a rich set of
+        comparisons to be performed based on a metadata category of interest. For
+        example, "where qiita_study_id == 10317" will find all samples which have
+        the qiita_study_id metadata category, and in which the value for that
+        sample is "10317."
+        <br/><br/>
+        Examples:
+        <br/>
+        <ul>
+          <li>
+            Find all samples in which the word infant exists, as well as antibiotics,
+            where the infants are under a certain number of days old:
+            "infant &amp; antibiotics where age_days < 30"
+          </li>
+          <li>
+            Find all samples only beloging to the EMP in which the ph in under 7:
+            "where ph < 7 &amp; emp_release1 == True"
+          </li>
+        </ul>
+      </li>
+      <li>
```

Comment:
So just to be clear, it's not possible to combine metadata searches with sequence searches? Like, I can find all ACGTTTGCCAGCAAAGACT in samples where ph < 7?

Reply:
It's possible with redbiom. Remember the command line interface is intended for piping, so this specific functionality may not be exposed to the CLI but is not be a problem with the API. The high level approach is:

- fetch the set of samples containing feature X
- fetch the set of samples corresponding to metadata criteria Y
- assess the set intersection of X and Y

Reply:
right, this might be possible by adding a GUI where you can "connect" searches but not sure it it will be easier to do this via redbiom directly ...